sc-deployer/index.ts

```diff
@@ -0,0 +1,51 @@
+import { Args, Client, ClientFactory, ProviderType, IAccount, WalletClient, IContractData, IProvider } from "@massalabs/massa-web3";
+import { readFileSync } from "fs";
+import path from "path";
+import { fileURLToPath } from 'url';
+
+export interface ISCData {
+    data: Uint8Array,
+    args?: Args,
+    coins: number,
+}
+
+const __filename = fileURLToPath(import.meta.url);
+
+const __dirname = path.dirname(__filename);
+
+async function deploySC(publicApi: string, account: IAccount, contracts: ISCData[], fee: number, maxGas: number) {
+    const client: Client = await ClientFactory.createCustomClient(
+        [
+            {url: publicApi, type: ProviderType.PUBLIC} as IProvider, 
+            // This IP is false but we don't need private for this script so we don't want to ask one to the user 
```

Comment:
what do you mean by 'this ip is false'?

Reply:
I use the same IP as the public one so with 33035 as port so it will not work for a private API but we don't care in our case.